README.md

```diff
@@ -32,27 +36,76 @@ pip install django-google-structured-logger
 
 #### Configuration
 
-1. Add `GoogleFormatter` to your Django's `LOGGING` setting.
-   Example:
+1. Add a formatter to your Django's `LOGGING` setting.
+
+   **For standard JSON logging:**
    ```python
     LOGGING = {
         "version": 1,
         "disable_existing_loggers": False,
         "formatters": {
             "json": {
-                "()": "django_google_structured_logger.formatter.GoogleFormatter",
+                "()": "django_google_structured_logger.formatter.StandardJSONFormatter",
             },
         },
         "handlers": {
             "console": {
                 "level": "INFO",
                 "class": "logging.StreamHandler",
             },
-            "google-json-handler": {
+            "json-handler": {
                 "class": "logging.StreamHandler",
                 "formatter": "json",
             },
         },
+        "root": {
+            "handlers": [env.str("DJANGO_LOG_HANDLER", "json-handler")],
+            "level": env.str("ROOT_LOG_LEVEL", "INFO"),
+        },
+        "loggers": {
+            "()": {
+                "handlers": [env.str("DJANGO_LOG_HANDLER", "json-handler")],
+                "level": env.str("DJANGO_LOG_LEVEL", "INFO"),
+            },
+            "django": {
+                "handlers": [env.str("DJANGO_LOG_HANDLER", "json-handler")],
+                "level": env.str("DJANGO_LOG_LEVEL", "INFO"),
+                "propagate": False,
+            },
+            "django.server": {
+                "handlers": [env.str("DJANGO_LOG_HANDLER", "json-handler")],
+                "level": env.str("DJANGO_SERVER_LEVEL", "ERROR"),
+                "propagate": False,
+            },
+            "django.request": {
+                "handlers": [env.str("DJANGO_LOG_HANDLER", "json-handler")],
+                "level": env.str("DJANGO_REQUEST_LEVEL", "ERROR"),
+                "propagate": False,
+            },
```

Comment:
Thinking further, I want to use a common logger for VarioSystems  FastAPI project. Is it compatible?